R/sel.R

```diff
@@ -85,7 +85,6 @@ m_sel <- function(
     bonds = bonds,
     elem = elem,
     atom = atom,
-    bonds = bonds,
```

Comment:
[Q] Why `bonds` is removed here?

Reply:
There was a double-up of bonds so I removed one of them.